samwise.js

```diff
@@ -1,52 +1,52 @@
 /**
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?